common/source/docs/common-accelerometer-calibration.rst

```diff
@@ -15,7 +15,7 @@ This article shows how to perform basic accelerometer calibration (using
 
 .. important::
 
-   Accelerometer calibration cannot be performed while vehicle is armed.
+   Accelerometer calibration cannot be performed while vehicle is armed. 
```

Comment:
Not sure why this whitespace change is needed.

Reply:
It was an accident. I wanted to revert my previous change as requested by @Hwurzburg. This is just me putting it back to what it was.